src/pages/o/[orgId]/events/[eventId].tsx

```diff
@@ -69,80 +82,80 @@ const OrgEventPage : PageWithLayout<OrgEventPageProps> = (props) => {
     const org = orgQuery.data as ZetkinOrganization;
 
     return (
-        <Flex direction="column">
-            <Heading data-test="event-title" level={ 1 }>
-                { event.title ? event.title : event.activity.title }
-            </Heading>
-            <Flex>
-                <Text marginEnd="size-50">
-                    <Msg id="pages.orgEvent.details.org"/>:
-                </Text>
+        <>
+            <Header marginBottom="size-300">
+                <Image
+                    alt="Cover image"
+                    height="size-2000"
+                    objectFit="cover"
+                    src="/cover.jpg"
+                    width="100%"
+                />
+                <Heading data-test="event-title" level={ 1 } marginBottom="size-50">
+                    { event.title ? event.title : event.activity.title }
+                </Heading>
                 <Link>
                     <NextLink href={ `/o/${orgId}` }>
                         <a>{ org.title }</a>
                     </NextLink>
                 </Link>
-            </Flex>
-            <Flex>
-                <Text marginEnd="size-50">
-                    <Msg id="pages.orgEvent.details.campaign"/>:
-                </Text>
+            </Header>
+            <Flex marginBottom="size-100">
+                <Flag aria-label="S Flag" marginEnd="size-100" size="S"/>
                 <Link>
                     <NextLink href={ `/o/${orgId}/campaigns/${event.campaign.id}` }>
                         <a>{ event.campaign.title }</a>
                     </NextLink>
                 </Link>
             </Flex>
-            <Flex>
-                <Text marginEnd="size-50">
-                    <Msg id="pages.orgEvent.details.start"/>:
-                </Text>
-                <Text data-test="start-time">
-                    <FormattedDate
-                        day="2-digit"
-                        month="long"
-                        value={ Date.parse(event.start_time) }
-                    />
-                    , <FormattedTime
-                        value={ Date.parse(event.start_time) }
-                    />
-                </Text>
+            <Flex alignItems="center" data-test="duration" marginBottom="size-100">
+                <Calendar aria-label="S Calender" marginEnd="size-100" size="S"/>
+                <Flex direction="column">
+                    <Text data-test="start-time">
```

Comment:
The `data-test` id doesn't seem to reflect the content here. Is the test ID necessary? If not, remove it, otherwise it should be changed to correctly reflect the content.